### PR TITLE
(CLU) Lonis, Genetics Expert

### PR DESCRIPTION
**Pull request by Kev-inBacon**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Lonis, Genetics Expert
+ManaCost:1 GU GU
+Types:Legendary Creature Snake Elf Detective
+PT:1/2
+K:Evolve
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigInvestigate | TriggerDescription$ Whenever one or more +1/+1 counters are put on CARDNAME investigate that many times.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt:6]: use NICKNAME

Comment (Kev-inBacon) [reply forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt]: Cool - didn't know that existed. Updated.

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Lonis, Genetics Expert
+ManaCost:1 GU GU
+Types:Legendary Creature Snake Elf Detective
+PT:1/2
+K:Evolve
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigInvestigate | TriggerDescription$ Whenever one or more +1/+1 counters are put on NICKNAME investigate that many times.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt:6]: missing "," there: `NICKNAME, investigate`

Comment (Kev-inBacon) [reply forge-gui/res/cardsfolder/upcoming/lonis_genetics_expert.txt]: Fixed.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**